src/js/standard/icons/volume.js

```diff
@@ -0,0 +1,35 @@
+import globalStyles from '../global-styles.scss';
+import createClassNameString from '../../lib/create-classname-string';
+
+export const volumeMuteIcon = (cssClass) =>`
```

Comment:
Well, my reasoning for using arrow functions instead of named functions is because I don't use `this` or it being inherital.